src/auth/Components/Footer.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import About from '../Components/About';
+import Cookies from '../Components/Cookies';
+import TermOfUse from '../Components/TermOfUse';
+import Privacy from '../Components/Privacy';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route
+} from "react-router-dom";
+
+const Footer = () =>(
+
+  <Container fluid style={{backgroundColor: '#212529', color: '#fff'}}>
```

Comment:
Here you need to use `next/link` instead of the react router. When each of the pages are moved to `pages/` directory, you will be able to link to them just like it is described in https://nextjs.org/docs/api-reference/next/link